config/routes.rb

```diff
@@ -1,3 +1,9 @@
 Rails.application.routes.draw do
-  resources :books, only: [:show]
+  resources :books, only: [:show] do
+    resources :reviews
+  end
+
+  resources :users do
+    resources :reviews
+  end
```

Comment:
Your user stories only refer to a book show, I would expect to only see `resources :books, only: [:show]`